tools/run_tests/xds_k8s_test_driver/framework/infrastructure/traffic_director.py

```diff
@@ -398,13 +407,9 @@ def _generate_url_map_body(
             }],
         }
 
-    def create_url_map(
-        self,
-        src_host: str,
-        src_port: int,
-    ) -> GcpResource:
-        src_address = f'{src_host}:{src_port}'
+    def create_url_map(self, src_host: str, src_port: int) -> GcpResource:
         name = self.make_resource_name(self.URL_MAP_NAME)
+        src_address = f'{src_host}:{src_port}'
```

Comment:
optional: We don't seem to need to change this file.

Reply:
changed by `yapf`